code/modules/clothing/under/chameleon.dm

```diff
@@ -81,3 +77,22 @@
 	for(var/U in typesof(/obj/item/clothing/under)-blocked)
 		var/obj/item/clothing/under/V = new U
 		src.clothing_choices += V
+
+/obj/item/clothing/under/chameleon/cold
+	heat_conductivity = 1000
+	var/registered_user = null
+
+/obj/item/clothing/under/chameleon/cold/attack_self(mob/user as mob)
+	if(!registered_user || registered_user == user)
+		if(!registered_user)
+			to_chat(usr, "You are registered as the user of this suit")
+			registered_user = user
+		if(!(/obj/item/clothing/under/chameleon/proc/Change_Color in verbs))
+			verbs |= /obj/item/clothing/under/chameleon/proc/Change_Color
+			return
+		if(/obj/item/clothing/under/chameleon/proc/Change_Color in verbs)
+			verbs &= ~/obj/item/clothing/under/chameleon/proc/Change_Color
+			return
+/obj/item/clothing/under/chameleon/cold/attackby(obj/item/clothing/under/U as obj, mob/user as mob)
+	if(registered_user == user)
```

Comment:
I may not be sure on how things work now but doesn't this make it so only the antag who 'owns' the suit can put like ties or medals on the suit now?